crates/node/core/src/args/debug.rs

```diff
@@ -26,6 +26,11 @@ pub struct DebugArgs {
     #[arg(long = "debug.max-block", help_heading = "Debug")]
     pub max_block: Option<u64>,
 
+    /// Flag indicating whether the node should send locally-produced payload
+    /// back to consensus engine for validation.
+    #[arg(long = "debug.payload-building-validation", help_heading = "Debug")]
+    pub payload_building_validation: bool,
```

Comment:
opt-in makes sense for eth, but for op this should always be enabled

Reply:
Forwarding payload is always enabled, this flag only controls whether the payload will be validated or not